### PR TITLE
Change routing to attributes

### PR DESCRIPTION
**Pull request by Nispeon**
Required doctrine/annotations and changed all routes from `config/routes.yaml` to controller attributes

### CODE REVIEW

**Comment by lyrixx** [comment]
>Required doctrine/annotations
Why ?

**Comment by Nispeon** [comment]
> > Required doctrine/annotations
> 
> Why ?

This dependency was necessary for the routing via attributes to work. It should not be necessary in PHP 8, but I could not get it to work without it and I've seen issues of people with the same problem.

File: src/Controller/SantaController.php [hunk 1]
```diff
@@ -317,6 +325,7 @@ public function cancel(Request $request, string $application): Response
         return $this->redirectToRoute('homepage');
     }
 
+    #[Route('/spoil', name:'spoil', methods:['GET'])]
```
Comment (pyrech) [inline src/Controller/SantaController.php:328]: The method POST is missing here, isn't it?

Comment (Nispeon) [reply src/Controller/SantaController.php]: You're right it was. I added it, thank you

**Review by pyrech — APPROVED** [review]

**Pull request merged**